spec/lita/handler/event_router_spec.rb

```diff
@@ -61,5 +71,41 @@ def greet(payload)
       subject.trigger(robot, "connected")
       subject.trigger(robot, " ConNected  ")
     end
+
+    it "doesn't stop triggering callbacks after an exception is raised" do
+      test_mode = Lita.test_mode?
+      begin
+        Lita.test_mode = false
+        expect(robot).to receive(:send_message).with("first error").once
+        expect(robot).to receive(:send_message).with("second error").once
+        subject.trigger(robot, :multiple_errors)
+      ensure
+        Lita.test_mode = test_mode
+      end
+    end
+
+    it "report callback exceptions to the error_handler" do
```

Comment:
Should be phrased imperatively: "reports callback exceptions to the error handler"